core/pom.xml

```diff
@@ -296,21 +285,25 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>org.springframework.security</groupId>
-      <artifactId>spring-security-acl</artifactId>
-      <version>${spring-security.version}</version>  <!-- see maven-parent-pom -->
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-aop</artifactId>
       <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.security</groupId>
+      <artifactId>spring-security-core</artifactId>
       <exclusions>
         <exclusion>
           <artifactId>*</artifactId>
           <groupId>*</groupId>
         </exclusion>
       </exclusions>
     </dependency>
+    <!-- -->
```

Comment:
I'm assuming this `<!-- -->`marker, marks the end of `test` scope dependencies.
You can use Intellij Idea regions to integrate better with our development tools: https://youtrack.jetbrains.com/issue/IDEA-93649
Just use `<region RegionName -->` and `<!-- endregion -->` to have folding directly available in the IDE.

Reply:
It makes sense, I will change in all PR's to follow this format.